easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-foss-2016a.eb

```diff
@@ -52,6 +63,8 @@ configopts += '-DWITH_IPP=ON '
 configopts += '-DENABLE_SSE=ON -DENABLE_SSE2=ON -DENABLE_SSE3=ON '
 configopts += '-DWITH_CUDA=OFF '
 
+postinstallcmds = ["cp 3rdparty/ippicv/unpack/ippicv_lnx/lib/intel64/libippicv.*  %(installdir)s/lib"]
```

Comment:
this should be reflected in `sanity_check_paths`? (also, remove the double space ;-))